vite.config.ts

```diff
@@ -1,10 +1,16 @@
 import react from "@vitejs/plugin-react";
 import { defineConfig } from "vite";
+import path from "path";
 
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [react()],
-  base: "/local-plans-timetable/",
+  base: "/local-plans-timetable",
```

Comment:
Removed the trailing slash because it meant that `/local-plans-timetable` wasn't a valid path